src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -58,7 +58,7 @@ public function find( WP_Block_Parser_Block $block ) {
 
 		}
 
-		return $strings;
+		return apply_filters( 'wpml_found_strings_in_block', $strings, $block );
```

Comment:
@kkarpieszuk, please add a PHP Doc complying with WP standards https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters. We sometimes forget to do it, but it's even more important with public packages.

Reply:
done